neuralmagic/tools/profiler/visualize_trace.py

```diff
@@ -14,7 +70,14 @@ def trim_string_back(string: str, width: int):
     return string
 
 
-def abbreviate_known_names(name: str):
+def shorten_plot_legend_strings(legend, max_char_len: int):
+    for t in legend.get_texts():
+        t.set_text(
+            trim_string_back(abbreviate_known_names(t.get_text()),
+                             max_char_len))
```

Comment:
thank you for this :)

Reply:
Lucas had this already :)

Reply:
I see now that it was there before this PR. The "thank you" still stands given how wide some of the previous plot legends were :)